stdlib/sys.mli

```diff
@@ -354,6 +354,19 @@ val runtime_warnings_enabled: unit -> bool
 
     @since 4.03.0 *)
 
+val enable_float_array_alloc_warning: int option -> unit
+(** Control whether the OCaml runtime system can emit warnings when values
+    of type [float array] arrays are allocated.
+    None   -> the warning is disabled
```

Comment:
I'm not sure how this ASCII art would render in the generated documentation.  It would be good to mention the constraint on [s], and the exception being generated when it isn't satisfied.